fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/impl/resource/loader/FabricModResourcePack.java

```diff
@@ -52,8 +52,7 @@ public InputSupplier<InputStream> openRoot(String... pathSegments) {
 			return () -> IOUtils.toInputStream(pack, Charsets.UTF_8);
 		} else if ("pack.png".equals(fileName)) {
 			return FabricLoader.getInstance().getModContainer("fabric-resource-loader-v0")
-					.flatMap(container -> container.getMetadata().getIconPath(512).map(container::getPath))
-					.filter(Files::exists)
+					.flatMap(container -> container.getMetadata().getIconPath(512).flatMap(container::findPath))
```

Comment:
Why is the `filter` call not necessary?

Reply:
See https://maven.fabricmc.net/docs/fabric-loader-0.14.10/net/fabricmc/loader/api/ModContainer.html#findPath(java.lang.String)